### DISCUSSION TITLE
language server suggestion only

### DISCUSSION THREAD

**Discussion by amgodoi** [neoclide/coc.nvim#3915] [Q&A] [answered]
Is there any way to exclude Buffer [B] and Around [A] from suggestions?

I would like to keep Language Server [LS] suggestions only.

**Comment by xiyaowong** [comment, marked as answer]
https://github.com/neoclide/coc.nvim/blob/43744ba82bd348ea971b87483a13f06f3e46c94b/data/schema.json#L1573

**Reply by amgodoi** [reply]
Hi!

    "coc.source.around.enable": false,
    "coc.source.buffer.enable": false,

Worked! Thanks a lot!

But, is it used anywhere other than code completion suggestion?

Is any other functionality lost?

**Reply by chemzqm** [reply]
> But, is it used anywhere other than code completion suggestion?

No

**Reply by amgodoi** [reply]
Thanks!
